homeassistant/components/zha/button.py

```diff
@@ -18,11 +18,15 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .core import discovery
+from .core.channels.base import ZigbeeChannel
```

Comment:
This can be moved inside the `TYPE_CHECKING` block

Reply:
done